src/plugins/kibana/public/discover/index.html

```diff
@@ -1,5 +1,53 @@
 <div ng-controller="discover" class="app-container">
-  <navbar name="discover">
+  <navbar name="discover-options">
+    <div class="discover-info">
+      <span ng-show="opts.savedSearch.id" class="discover-info-title">
+        <span ng-bind="::opts.savedSearch.title"></span>
+        <i aria-label="Reload Saved Search" tooltip="Reload Saved Search" ng-click="resetQuery();" class="fa fa-undo small"></i>
+      </span>
+
+      <strong class="discover-info-hits">{{(hits || 0) | number:0}}</strong>
+      <ng-pluralize count="hits" when="{'1':'hit', 'other':'hits'}"></ng-pluralize>
+    </div>
+    <div class="discover-actions button-group" role="toolbar">
```

Comment:
`discover-actions` should be a general actions class, or simply a style for button groups in this sort of nav bar, no reason for it to be special